blueprints/docker-for-mac.yml

```diff
@@ -79,6 +79,9 @@ services:
             "--swarm-default-advertise-addr=eth0",
             "--userland-proxy-path", "/usr/bin/vpnkit-expose-port",
             "--storage-driver", "overlay2" ]
+  # Monitor for image deletes and invoke a TRIM on the container filesystem
+  - name: trim-after-delete
+    image: "test"
```

Comment:
everything's set up - @djs55 since I don't have your key yet (we can sort this out later), just ping me when you've addressed all review comments for the package and I'll push/sign